erpnext_com/www/docs/user/manual/en/loan-management/loan-application.md

```diff
@@ -0,0 +1,50 @@
+<!-- add-breadcrumbs -->
+# Loan Application
+> Introduced in Version 13
+
+**A Loan Application contains details about the applicant and loan security details for review.**
+
+During a loan process, the first step a customer or employee has to do is submit a loan application for review. In case of a secured loan, a loan application can also contain proposed loan securities.
+
+<img class="screenshot" alt="Make Loan Application" src="{{docs_base_url}}/assets/img/loan-management/loan-application-flow.png">
+
+To access the Loan Application list, go to:
+> Home > Loan Management > Loan > Loan Application
+
+
+<img class="screenshot" alt="Loan Application" src="{{docs_base_url}}/assets/img/loan-management/loan-application.png">
+
+## 1. Prerequisites
+Before creating and using a Loan Security, it is advised that you create the following first:
+
+* [Loan Security Type](/docs/user/manual/en/loan-management/loan-security-type)
+* [Loan Security](/docs/user/manual/en/loan-management/loan-security)
+* [Loan Type](/docs/user/manual/en/loan-management/loan-type)
+
+## 2. How to Create a Loan Application
+1. Go to Loan Application List, click on New.
+2. Select the applicant type.
+3. Select the applicant.
+4. Select the Loan Type.
+5. Enter the Loan Amount.
```

Comment:
fields like rate of interest, repayment period in months, total payable interest not explained
what are the other options in Repayment Method?? docs need to have complete info

Reply:
@prasadarr There are only two repayment methods

Reply:
@deepeshgarg007 fields like rate of interest, repayment period in months, total payable interest not explained. docs need to have complete info